examples/gitops/argocd-multi-cluster/hub-cluster/main.tf

```diff
@@ -0,0 +1,372 @@
+provider "aws" {
+  region  = local.region
+  profile = local.hub_profile
```

Comment:
why do we need a profile here? this is tied to a users local configuration - if we need to go cross account, I think role assumption is the preferred approach and more reliable

Reply:
Do you have examples how to best do cross accounts? so far I was able to get it working with local profiles